apps/vaporgui/MainForm.cpp

```diff
@@ -1712,12 +1885,20 @@ void MainForm::_setAnimationOnOff(bool on) {
 		enableAnimationWidgets(false);
 
 		_App->removeEventFilter(this);
+//        if (_progressEnabled) {
```

Comment:
If this code might be needed later, please ifdef and provide a comment. Otherwise delete.

Reply:
Done